### DISCUSSION TITLE
Upgrading MLB-StatsAPI

### DISCUSSION THREAD

**Discussion by smith288** [WardBrian/mlb-led-scoreboard#19] [Q&A] [answered]
So Im not sure what other steps I need to make here

![image](https://user-images.githubusercontent.com/68360/186686068-0166f61c-7640-43a5-937b-6fc5be2ebf99.png)

Can you assist in what i am missing?

**Comment by tracerrx** [comment, marked as answer]
It looks like your running Python 2.7, Try Python3

**Reply by smith288** [reply]
Thank you so much... I cant believe I didnt notice that.  Apparently, my python is still wired to 2.7.  running the **sudo python3 -m pip install --upgrade MLB-StatsAPI** is the solution for me.
